### DISCUSSION TITLE
Date of articles - when to show a post based off the date

### DISCUSSION THREAD

**Discussion by dwelsh-clouru** [lxndrblz/anatole#133] [Q&A]
Hi There,

I am trying to figure out what I need to change in order to post an article in the future / where the "if _date_ **<** today()" parameter is...? struggling a little bit to find it. If anyone could point me in the right direction, that would be fantastic!

Cheers.

**Comment by lxndrblz** [comment]
@admin-clouru Hi there, this functionality is currently not implemented. Why you could do is to flag is a draft and then it wouldn't get published immediately. Your today comparison function will only make sense if you build your site daily. Then you could add the necessary check to the index.html template. You would have to extend the paginate function with a condition for checking the dates. Let me know if that is what you really want?

**Reply by dwelsh-clouru** [reply]
Hi There,

Thanks for the response! Not sure if I was clear in my question...  At the moment, If I publish an article for today, it will not show up until tomorrow. However, I want every article to be shown regardless of the date!

let me know if I need to make myself clearer / add examples / screenshots .. Thanks!

**Reply by lxndrblz** [reply]
@admin-clouru Thanks for clarifying. Would you mind sharing an example and how you usually publish, because I don't have this issue on my Netlify setup.

Whatever I submit today is visible instantly.

**Reply by dwelsh-clouru** [reply]
My suspicion is that It could be because I am in Australia, and therefore there is a delayed effect because of the time difference. 

I assume that you would be able to replicate this by creating a blog post with a future date?

I publish a post by simply creating a new markdown file within the content folder and specify the meta data as per below:

```
+++
author = "Daniel Welsh"
title = "variable summers day"
date = "2021-02-04"
draft = false
description = "Blog Post"
+++
```

**Reply by lxndrblz** [reply]
You could try to specify the date including time and time zone:

`date: 2021-02-08T14:15:59+11:00`

I will try to create a future post later on and report back.
